pq/ack.go

```diff
@@ -250,10 +248,12 @@ func (a *acker) collectFreePages(c *txCursor, endID uint64) ([]txfile.PageID, bo
 
 // findNewStartPositions skips acked events, so to find the new head and read pointers to be set
 // in the updated queue header.
-func (a *acker) findNewStartPositions(c *txCursor, id uint64) (head, read position, err error) {
+func (a *acker) findNewStartPositions(c *txCursor, id uint64) (head, read position, err reason) {
+	const op = "pq/ack-compute-new-start"
+
 	var hdr *eventPage
 
-	hdr, err = c.PageHeader()
+	hdr, err = c.PageHeader(op)
```

Comment:
Why cannot simply `errWrap()` instead of passing the current op?

Reply:
I am not sure I understand the need to pass around the current operation to other method? I did a quick look at the other methods receiving the operation and I think we could still wrap in the caller instead?

Reply:
Ah, will change it to use errWrap. Code passing `op` is somewhat older, before I introduced errWrap and context merging.